### ISSUE TITLE
SBT search still seems to be broken

### ISSUE THREAD

**Issue by ctb** [dib-lab/sourmash#454]
I recalculated the genbank-k31 SBT and have not been able to find things in it. Code from #244 needs to be revisited.

**Comment by MrCreosote** [comment]
If it helps, I just reproduced this problem on our own data. When I run a query that is identical to a leaf in the SBT, I get no results:

```
$ date; sourmash search ~/kb_refseq_sourmash/kb_refseq_ci_1000_15792_446_1.sig kb_refseq_ci.sbt.json; date
Mon Apr 23 11:52:34 PDT 2018
select query k=31 automatically.
loaded query: 15792_446_1... (k=31, DNA)
loaded 1 databases.

0 matches:
similarity   match
----------   -----
Mon Apr 23 13:10:17 PDT 2018


$ diff .sbt.kb_refseq_ci/da3a39a1e4b05acac89c380731b7c4b3 ~/kb_refseq_sourmash/kb_refseq_ci_1000_15792_446_1.sig 
$ 

$ sourmash info
sourmash version 2.0.0a4
- loaded from path: /usr/local/lib/python2.7/dist-packages/sourmash

```

**Comment by ctb** [comment]
I've been able to find a reproducible bug with public data (!!), using the new utils/check-tree code in #484:

1. Download and unpack https://osf.io/8uxj9/
2. `sourmash compute -k 21,31,51 --name-from-first --scaled=1000 *.fa`
3. Build index `sourmash index -k 31 pg *.sig`
4. Run `check-tree.py`: `python ../utils/check-tree.py pg`

and then I see that N. equitans, 33.fa, isn't found in the tree; double check with

```
sourmash search 33.fa.sig pg
```

**Comment by ctb** [comment]
Pretty sure I fixed this with a merged PR since June but ...failed to link this issue to fix? #willfix

**Comment by ctb** [comment]
Yes, fixed in #484.
